Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_ValidateSPIManifest_show.1.html

```diff
@@ -77,7 +77,7 @@ <h2 class="trimmed">Validate a Swift Package Index manifest</h2>
               <p>Enter the contents of a 
                 <code>.spi.yml</code> file for validation:
               </p>
-              <textarea name="manifest" autofocus rows="15" maxlength="1000">version: 1
+              <textarea name="manifest" autofocus rows="15" maxlength="1600">version: 1
```

Comment:
1600 or 1500? Is this so there's a chance for the error to show?

We should have the error message server-side in case anything breaks or is being bypassed on the client-side, but I think if we only allow 1,500 bytes, we should limit this to 1,500 characters.

Reply:
Can we also drive this from the constant?

Reply:
That's intentional so we can actually detect an input that's too large. `maxlength` clips the input string, doesn't it? If we clip it here, we won't give `Manifest.load` a chance to detect that it's over size.